genie-web/src/main/java/com/netflix/genie/web/agent/services/impl/AgentJobServiceImpl.java

```diff
@@ -124,6 +126,17 @@ public void handshake(
 
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Map<String, String> getAgentProperties(
+        @Valid final AgentClientMetadata agentClientMetadata
+    ) {
+        // TODO: currently NOOP. Implement.
```

Comment:
I'm going in order of commits so I'll assume this is completed in another commit?